client/src/cmdhficlass.c

```diff
@@ -5403,18 +5404,66 @@ static int CmdHFiClassSAM(const char *Cmd) {
     void *argtable[] = {
         arg_param_begin,
         arg_lit0("v", "verbose", "verbose output"),
+        arg_lit0("k", "keep", "keep the field active after command executed"),
+        arg_lit0("n", "nodetect", "skip selecting the card and sending card details to SAM"),
+        arg_lit0("t",  "tlv",      "decode TLV"),
+        arg_lit0(NULL, "break-on-nr-mac", "stop tag interaction on nr-mac"),
+        arg_lit0("p", "prevent-epurse-update", "fake epurse update"),
+        arg_lit0(NULL, "shallow", "shallow mod"),
+        arg_strx0("d", "data",     "<hex>", "DER encoded command to send to SAM"),
         arg_param_end
     };
     CLIExecWithReturn(ctx, Cmd, argtable, true);
-    bool verbose = arg_get_lit(ctx, 1);
+
+    bool verbose = false;
+    if (arg_get_lit(ctx, 1)) {
+        verbose = true;
+    }
```

Comment:
you can simplify all these if-statements,  to just an assignment.
Like this. 
`bool verbose = arg_get_lit(ctx, 1);`